package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@usecanvas/canvas-editor",
+  "name": "@rebelworks/canvas-editor",
```

Comment:
ah, my bad. i was planning do some heavy experiments / modification to canvas-editor, so i created a new npm package for convenience.